Samples/outlook-encrypt-attachments/README.md

```diff
@@ -0,0 +1,188 @@
+
+Once the add-in is loaded use the following steps to try out the functionality:
+
+#### Manage attachments
+1. Create a new message or appointment
+    > You should see a notification at the top of the message that reads: **Open the task pane for details about running the Outlook event-based activation sample add-in.** | Open task pane | Dismiss
+2. Add an attachment
+    > You should see a notification at the top of the message that reads: **The '{file name} attachment has been encrypted and decrypted and added as reference attachments for your review** | Dismiss
+
+#### Set up meetings
+1. Create a new meeting request. In Outlook on the web, choose **More options** to expand the request and include all details (otherwise you won't see the notifications in the next steps.)
+2. Add a user as a required or optional attendee
+    > You should see a notification at the top of the message that reads: **Your appointment has 1 required and 0 optional attendees** | Dismiss
+3. Add a distribution list as a required or optional attendee
+    > You should see a notification at the top of the message that reads: **Warning! Your appointment has a distribution list! Make sure you have chosen the correct one!** | Dismiss
+4. Change the start or end date or time
+    > You should see a notification at the top of the message that reads: **Original date/time: Start = ##/##/#### #:##:## ##; End = ##/##/#### #:##:## ##** | Dismiss
+
+## Run the sample from localhost
+
+If you prefer to host the web server for the sample on your computer, follow these steps:
+
+1. Install a recent version of [npm](https://www.npmjs.com/get-npm) and [Node.js](https://nodejs.org/) on your computer. To verify if you've already installed these tools, run the commands `node -v` and `npm -v` in your terminal.
+1. You need http-server to run the local web server. If you haven't installed this yet, run the following command.
+
+    ```console
+    npm install --global http-server
+    ```
+
+1. Use a tool such as openssl to generate a self-signed certificate that you can use for the web server. Move the cert.pem and key.pem files to the root folder for this sample.
+1. From a command prompt, go to the root folder and run the following command.
+
+    ```console
+    http-server -S --cors . -p 3000
+    ```
+
+1. To reroute to localhost, run office-addin-https-reverse-proxy. If you haven't installed this, run the following command.
+
+    ```console
+    npm install --global office-addin-https-reverse-proxy 
+    ```
+
+    To reroute, run the following in another command prompt.
+
+    ```console
+    office-addin-https-reverse-proxy --url http://localhost:3000 
+    ```
+
+1. Sideload `manifest.xml` in Outlook on the web or on Windows by following the manual instructions in the article [Sideload Outlook add-ins for testing](https://docs.microsoft.com/office/dev/add-ins/outlook/sideload-outlook-add-ins-for-testing).
+1. [Try out the sample!](#try-it-out)
+
+## Key parts of this sample
```

Comment:
This seems unnecessary. Aren't all parts of the sample "key parts?" Consider removing this H2 and then elevating the following H3s to H2 which will add them to the right pane links.